### ISSUE TITLE
Try to add through inline doesnt save the json

### ISSUE THREAD

**Issue by flp9001** [bhch/django-jsonform#27]
If I try to add a new related object that has a jsonform through an inline it doesn't save the options. 
If I edit it works as expected.

models.py:

```python
from django.db import models

from django_jsonform.models.fields import JSONField


PREF_SCHEMA = {
    "type": "dict",
    "keys": {
        "periodTime": {
            "title": "Time of day",
            "type": "dict",
            "keys": {
                "morning": {
                    "type": "boolean",
                    "title": "Morning",
                },
                "afternoon": {
                    "type": "boolean",
                    "title": "Afternoon",
                },
                "evening": {
                    "type": "boolean",
                    "title": "Evening",
                },
            },
        },
    },
}


class MyModel(models.Model):
    name = models.CharField(max_length=50, null=True, blank=True)

class MyRelatedModel(models.Model):
    pref = JSONField(schema=PREF_SCHEMA)
    mymodel = models.ForeignKey(MyModel, on_delete=models.CASCADE)

```

admin.py:
```python
from django.contrib import admin
from django import forms

from .models import MyModel
from .models import MyRelatedModel



class MyRelatedModelForm(forms.ModelForm):
    class Meta:
        model = MyRelatedModel
        fields = ("pref",)



class MyRelatedModelInline(admin.TabularInline):
    model = MyRelatedModel
    form = MyRelatedModelForm
    extra = 0
    can_delete = True



@admin.register(MyModel)
class MyModelAdmin(admin.ModelAdmin):
    inlines = (MyRelatedModelInline,)


@admin.register(MyRelatedModel)
class MyRelatedModelAdmin(admin.ModelAdmin):
    pass


```

Here is a full project with that issue happening:

[proj.zip](https://github.com/bhch/django-jsonform/files/8043884/proj.zip)

**Comment by bhch** [comment]
There are two problems here:

First is that the widget requires the `id` of the JSON field where it will keep the data. Since that field is created dynamically, the `id` is also generated dynamically. However, the widget is initialized in a `script` tag ([see source](https://github.com/bhch/django-jsonform/blob/master/django_jsonform/templates/django_jsonform/editor.html#L6-L21)), but `id` values in the JS variables aren't updated.

This can be fixed easily by reading the `id` of the newly created element, however there's a second problem preventing that from happening.

Django uses jQuery's `clone()` function to create new inlines. The `script` tag also gets cloned, but the trouble is that [jQuery marks those cloned scripts as *already executed*](https://jquery.com/upgrade-guide/1.9/#loading-and-running-scripts-inside-html-content), hence the browser doesn't execute them again. That means we can't read the new id values and the widget can't find the JSON field to keep the data in.

I think a possible solution is to listen for events when a new inline is added and then manually execute the `script` tag. I'll see what be done.

**Comment by bhch** [comment]
I've fixed this in [version 2.8.0](https://github.com/bhch/django-jsonform/releases/tag/v2.8.0). Please upgrade.

Thank you.
